src/legacy/core_plugins/console/common/text_object.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export const type = 'text-object';
```

Comment:
I confess, my first thought on seeing this was, "oh, that's going to confused with the `type` keyword" 😁

Reply:
We had a [conversation](https://github.com/elastic/elasticsearch-team/issues/297) around trying to write more JavaDoc in the Elasticsearch codebase, and so I'm wondering if we can try to do the same with JSDoc. Specifically, maybe it would be useful to briefly document here what the purpose is of `TextObject`? Just a thought.

Reply:
Bah, this comment is supposed to be attached to `TextObject`